doc/release-notes.md

```diff
@@ -1,6 +1,6 @@
-Dogecoin Core version 1.14.2 is now available from:
+Dogecoin Core version 1.14.3 is now available from:
 
-  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.0/>
+  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.3/>
```

Comment:
link does not lead anywhere

Reply:
Cyclic update issue - we need to have link it will be at, in the readme before we create the tag to make the link valid.